lib/load/release-rules.js

```diff
@@ -28,9 +28,9 @@ module.exports = ({releaseRules}) => {
         throw new Error('Error in commit-analyzer configuration: rules must be an object with a "release" property');
       } else if (RELEASE_TYPES.indexOf(rule.release) === -1) {
         throw new Error(
-          `Error in commit-analyzer configuration: "${rule.release}" is not a valid release type. Valid values are: ${JSON.stringify(
-            RELEASE_TYPES
-          )}`
+          `Error in commit-analyzer configuration: "${
+            rule.release
+          }" is not a valid release type. Valid values are: ${JSON.stringify(RELEASE_TYPES)}`
```

Comment:
this is the weirdest update :D I don’t even get the error: https://travis-ci.org/semantic-release/commit-analyzer/builds/298356397#L468 do you know ... why?

Reply:
Due to https://github.com/prettier/prettier/pull/3124

It's indeed a bit weird. But it was weird before. The problem with long interpolated strings is that there is rarely a good way to display them. They either overflow or have to be broken somewhere...

Reply:
It took me a while to read it as well.
The error reported is basically "Replace `current` by `fixed`". 
But the string that has to be formatted contains "is not a valid" and "Valid values are" which make it confusing. it feel like those are part of the error explanation, but they are actually part of the `current` and `fixed`.